docutron/bin/helpers/extend-webpack-config.js

```diff
@@ -0,0 +1,35 @@
+const path = require( 'path' );
+
+module.exports = function( webpackConfig, usersCwd ) {
+	const docsFolder = 'docs';
```

Comment:
Should this be hard-coded as `docs`, or should we support specifying this from the command-line?